lib/lib_vty.c

```diff
@@ -34,12 +34,26 @@
 #if defined(HAVE_MALLINFO2) || defined(HAVE_MALLINFO)
 static int show_memory_mallinfo(struct vty *vty)
 {
+	char buf[MTYPE_MEMSTR_LEN];
```

Comment:
isn't this code change independent of using tcmalloc?  If so let's just move it into it's own commit?

Reply:
ok, will do